app/controllers/admin/ip_blocks_controller.rb

```diff
@@ -5,7 +5,7 @@ class IpBlocksController < BaseController
     def index
       authorize :ip_block, :index?
 
-      @ip_blocks = IpBlock.order(ip: :asc).page(params[:page])
+      @ip_blocks = IpBlock.order(ip: :asc).includes(:user).page(params[:page])
```

Comment:
Something is wrong here, `IpBlock` doesn't have a user association.

Reply:
Hmm -- this all came out of cherry pick from larger branch that adds bullet to help find these ... let me double check I didn't screw up a rebase or something for the ones in this PR.